sdk/communication/communication-tiering/swagger/swagger.json

```diff
@@ -0,0 +1,562 @@
+{
```

Comment:
Any reason why this is here instead of https://github.com/Azure/azure-rest-api-specs?

Reply:
Mainly because all the other sdks follow this pattern, wanted it to be consistent in the event we make changes to all of them, don't want added complexity.

Reply:
Gotcha, I just double checked and noticed that this is intended for private/internal use. In that case, this is indeed the pattern for the other SDKs in communication. However, for any public SDK, the corresponding OpenAPI swagger should be published in the Azure REST API specs repo.